src/backend/TrafficCourts/Citizen.Service/Features/Disputes/Create.cs

```diff
@@ -12,51 +12,34 @@ namespace TrafficCourts.Citizen.Service.Features.Disputes
 {
     public static class Create
     {
-        public class Request : IRequest<Response>
+        public class Request : IRequest
         {
-            public Models.Dispute.NoticeOfDispute Dispute { get; init; }
+            public NoticeOfDispute Dispute { get; init; }
 
             public Request(Models.Dispute.NoticeOfDispute dispute)
             {
                 Dispute = dispute ?? throw new ArgumentNullException(nameof(dispute));
             }
         }
 
-        public class Response
-        {
-            public Guid Id { get; init; }
-            public Exception? Exception { get; init; }
-
-            public Response(Guid id)
-            {
-                Id = id;
-            }
-
-            public Response(Exception exception)
-            {
-                Id = Guid.Empty;
-                Exception = exception;
-            }
-        }
-
-        public class CreateDisputeHandler : IRequestHandler<Request, Response>
+        public class CreateDisputeHandler : IRequestHandler<Request>
```

Comment:
Is there no error handling when creating a dispute? What if there is an error? The controller doesn't need to know?

Reply:
Added Response object back to handle success and error scenarios for create dispute handler as well as updated DisputeController to return different response codes according to the Response from the handler.